tests/phpunit/data/blocks/fixtures/core__gallery__deprecated-5.server.html

```diff
@@ -0,0 +1,48 @@
+
+<figure class="wp-block-gallery columns-3 is-cropped wp-block-gallery-1">
+	<ul class="blocks-gallery-grid">
+		<li class="blocks-gallery-item">
+			<figure>
+				<a href="http://wptest.local/wp-content/uploads/2020/09/test-image-edited-1-scaled.jpg" >
+					<img
+						src="http://wptest.local/wp-content/uploads/2020/09/test-image-edited-1-682x1024.jpg"
+						alt=""
+						data-id="705"
+						data-full-url="http://wptest.local/wp-content/uploads/2020/09/test-image-edited-1-scaled.jpg"
+						data-link="http://wptest.local/classic/test-image-3/"
+						class="wp-image-705"
+					/>
+				</a>
+			</figure>
+		</li>
+		<li class="blocks-gallery-item">
+			<figure>
+				<a href="http://wptest.local/wp-content/uploads/2020/09/test-image-edited-scaled.jpg" >
+					<img
+						src="http://wptest.local/wp-content/uploads/2020/09/test-image-edited-1024x682.jpg"
+						alt=""
+						data-id="704"
+						data-full-url="http://wptest.local/wp-content/uploads/2020/09/test-image-edited-scaled.jpg"
+						data-link="http://wptest.local/test-image-2/"
+						class="wp-image-704"
+					/>
+				</a>
+			</figure>
+		</li>
+		<li class="blocks-gallery-item">
+			<figure>
+				<a href="http://wptest.local/wp-content/uploads/2020/04/test-image-scaled.jpg" >
+					<img
+						src="http://wptest.local/wp-content/uploads/2020/04/test-image-1024x683.jpg"
+						alt=""
+						data-id="703"
+						data-full-url="http://wptest.local/wp-content/uploads/2020/04/test-image-scaled.jpg"
+						data-link="http://wptest.local/test-image/"
+						class="wp-image-703 is-layout-flex"
```

Comment:
`is-layout-flex` doesn't look appropriate here as it's not on the other images. I am unsure if it has an effect.